packages/internal/src/__tests__/paths.test.ts

```diff
@@ -890,78 +890,87 @@ describe('paths', () => {
 
     describe('processPagesDir', () => {
       it('it accurately finds and names the pages', () => {
-        const pagesDir = path.resolve(
-          __dirname,
-          '../../../../__fixtures__/example-todo-main/web/src/pages'
-        )
+        const pagesDir = path.join(FIXTURE_BASEDIR, 'web', 'src', 'pages')
 
         const pages = processPagesDir(pagesDir)
 
-        expect(pages.length).toEqual(8)
-
-        const adminEditUserPage = pages.find(
-          (page) => page.importName === 'adminEditUserPage'
-        )
-        expect(adminEditUserPage).not.toBeUndefined()
-        expect(adminEditUserPage.importPath).toEqual(
-          importStatementPath(
-            path.join(pagesDir, 'admin/EditUserPage/EditUserPage')
+        expect(pages.length).toEqual(20)
+
+        const pageNames = [
+          'AboutPage',
+          'BlogPostPage',
+          'ContactPage',
+          'FatalErrorPage',
+          'ForgotPasswordPage',
+          'HomePage',
+          'LoginPage',
+          'NotFoundPage',
+          'ProfilePage',
+          'ResetPasswordPage',
+          'SignupPage',
+          'WaterfallPage',
+        ]
+
+        for (let i = 0; i < pageNames.length; i++) {
```

Comment:
100% nitpicking here, but we'd generally use `pageNames.forEach((pageName) => { ...`

Reply:
More than happy to adopt the preferred choice